pcsx2/windows/VCprojects/pcsx2.vcxproj

```diff
@@ -625,4 +629,4 @@
   </ItemGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets" />
-</Project>
+</Project>
```

Comment:
Well, I think Visual Studio removes the newline if the project file is updated via the UI. Editing the files via the UI can really suck though (it can cause the files to bloat excessively, especially if you're changing compile settings). I think the newlines might be there cause I partially edited the files with vim when I did #1462 and that adds a newline to the end of file if it's absent.